### PR TITLE
Refactor genesis builder

### LINKED ISSUE #714: [Dev] Add build.rs file to generate the genesis and RPC spec file
Add build.rs file to generate the genesis and RPC spec file.

### PR DESCRIPTION
**Pull request by jolestar**
## Summary

1. use build.rs to generate stdlib and error code map files.
2. use build.rs to generate RPC spec file.
3. remove stdlib and error code map file from the source code repo.
4. pickup the commit from #723 

- Closes #714

### CODE REVIEW

File: crates/rooch-framework/sources/crypto/ecdsa_k1.move [hunk 1]
```diff
@@ -2,9 +2,9 @@ module rooch_framework::ecdsa_k1 {
     use std::vector;
 
     /// constant codes
-    const VALID_ECDSA_K1_TO_SCHEME_BITCOIN_LENGTH: u64 = 1;
-    const VALID_ECDSA_K1_COMPRESSED_PUBKEY_LENGTH: u64 = 33;
-    const VALID_ECDSA_K1_SIG_LENGTH: u64 = 64;
+    const ECDSA_K1_TO_SCHEME_BITCOIN_LENGTH: u64 = 1;
```
Comment (baichuan3) [inline crates/rooch-framework/sources/crypto/ecdsa_k1.move:5]: So, the convention for final constants is to choose without any prefix?

**Comment by baichuan3** [comment]
Does genesis generated file also removed from the source code repo? I don’t see the genesis generated file in the commit.

File: crates/rooch-framework/sources/crypto/ecdsa_k1.move (hunk 1 above)
Comment (jolestar) [reply crates/rooch-framework/sources/crypto/ecdsa_k1.move:5]: error code with `Error` prefix

**Comment by jolestar** [comment]
> Does genesis generated file also removed from the source code repo? I don’t see the genesis generated file in the commit.

The genesis file has already been removed in the #719

File: crates/rooch-framework/sources/crypto/ecdsa_k1.move (hunk 1 above)
Comment (baichuan3) [reply crates/rooch-framework/sources/crypto/ecdsa_k1.move:5]: I mean CONST prefix

Comment (jolestar) [reply crates/rooch-framework/sources/crypto/ecdsa_k1.move:5]: Do not need the `CONST` prefix for other `const`.

**Review by baichuan3 — APPROVED** [review]

**Pull request merged**